### PR TITLE
Created Grant migration

### PR DESCRIPTION
**Pull request by christianrd**
## What's new

According to #33, we have created **Grant** migration.

### CODE REVIEW

File: TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace DatabaseMigration.Migrations
+{
+    [Migration(20180602204300)]    
+    public class _20180602204300_create_grant_table : SoftDeleteEntityMigration<Grant>
+    {
+        public override void Up()
+        {
+            base.Up();
+            CreateForeignKeyColumn<Roles>(nameof(Entity.RoleId));
+            CreateForeignKeyColumn<Module>(nameof(Entity.ModuleId));
```
Comment (Willjobs94) [inline TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs:14]: Que representa el Module ?

Comment (christianrd) [reply TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs:14]: Eso vienen siendo los módulos. Ejemplo: Usuarios, clientes...

Comment (Willjobs94) [reply TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs:14]: Do a rebase with develop and I will merge it

Comment (Willjobs94) [reply TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs:14]: I will rebase it

Comment (christianrd) [reply TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs:14]: Ok

**Pull request merged**